projects/demo/src/app/app.routes.ts

```diff
@@ -136,8 +136,9 @@ export const appRoutes: Routes = [
 @NgModule({
     imports: [
         RouterModule.forRoot(appRoutes, {
-            initialNavigation: 'enabled',
+            initialNavigation: 'enabledBlocking',
             relativeLinkResolution: 'corrected',
+            scrollPositionRestoration: `enabled`,
```

Comment:
This option will be the default in the future Angular.

Reply:
please, also add to `taiga-ui.dev` that option

Reply:
@splincode okey, I will add it to `taiga` and `angular-open-source-starter`